community/en-us/development/subscribe.md

```diff
@@ -4,8 +4,20 @@ It is highly recommended to subscribe to the development mailing list to keep up
 
 In the process of using DolphinScheduler, if you have any questions or ideas, suggestions, you can participate in the DolphinScheduler community building through the Apache mailing list. Sending a subscription email is also very simple, the steps are as follows:
 
-1. Send an email to dev-subscribe@dolphinscheduler.apache.org with your own email address, subject and content.
+1. Send an email to dev-subscribe@dolphinscheduler.apache.org with your own email address, subject and content are arbitrary.
 
 2. Receive confirmation email and reply. After completing step 1, you will receive a confirmation email from dev-help@dolphinscheduler.apache.org (if not received, please confirm whether the email is automatically classified as spam, promotion email, subscription email, etc.) . Then reply directly to the email, or click on the link in the email to reply quickly, the subject and content are arbitrary.
 
 3. Receive a welcome email. After completing the above steps, you will receive a welcome email with the subject WELCOME to dev@dolphinscheduler.apache.org, and you have successfully subscribed to the Apache DolphinScheduler mailing list.
+
+# Unsubscribe Mailing Lists
+
+If you no longer need to know about DolphinScheduler's dynamics, you can unsubscribe from the mailing list
```

Comment:
I think this sentence should change, is a little odd. WDYT?

Reply:
How about change to:  If you do not need to know what's going on with DolphinScheduler, you can unsubscribe from the mailing list

Reply:
> If you do not need to know what's going on with DolphinSc

I think it's better, good